modules/social_features/social_landing_page/templates/node--featured.html.twig

```diff
@@ -105,53 +107,83 @@
     <div class="teaser__content">
 
       {% block card_title %}
-
         {{ title_prefix }}
-        {% if not page %}
           <h4{{ title_attributes }} class="teaser__title">
             <a href="{{ url }}" rel="bookmark">{{ label }}</a>
```

Comment:
Didn't really get this. Here we use the same approach as in all other templates.
I just removed `{% if not page %}` condition, since it doesn't make any sense here.